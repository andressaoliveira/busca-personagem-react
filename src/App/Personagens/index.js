import { connect } from "react-redux";
import Personagens from './Personagens';
import { listarPersonagens } from '../../Infraestructure/Ducks/PersonagensDuck';

export const mapStateToProps = state => {
    const { personagens } = state;
    return { personagens };
};

export const mapDispatchToProps = dispatch => ({
    listarPersonagens: () => {
        dispatch(listarPersonagens())
    },
});

export default connect(
    mapStateToProps,
    mapDispatchToProps
)(Personagens);