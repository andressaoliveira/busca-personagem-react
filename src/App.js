import Personagens from './Personagens/Personagens'
import './App.scss';

function App() {
  return (
    <div className="App">
      <Personagens />
    </div>
  );
}

export default App;
