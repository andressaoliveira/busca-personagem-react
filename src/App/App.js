import React from 'react';
import './App.scss';
import Personagens from './Personagens';

function App() {
  return (
    <div className="App">
      <Personagens />
    </div>
  );
}

export default App;
