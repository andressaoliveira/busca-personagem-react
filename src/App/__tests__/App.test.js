import expect from "expect";
import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';

import App from "../index";

configure({ adapter: new Adapter() })

describe("App", () => {
  function setup() {
    const props = {};
    return shallow(<App {...props} />);
  }

  const wrapper = setup();

  it("deve montar o componente sem deixar de funcionar", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
