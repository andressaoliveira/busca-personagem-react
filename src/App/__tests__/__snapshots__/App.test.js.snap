// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App deve montar o componente sem deixar de funcionar 1`] = `ShallowWrapper {}`;
